test/common/upstream/health_checker_impl_test.cc

```diff
@@ -226,8 +226,9 @@ TEST_F(HttpHealthCheckerImplTest, Success) {
   setupNoServiceValidationHC();
   EXPECT_CALL(*this, onHostStatus(_, false)).Times(1);
 
-  cluster_->hosts_ = {HostSharedPtr{new HostImpl(
-      cluster_->info_, "", Network::Utility::resolveUrl("tcp://127.0.0.1:80"), false, 1, "")}};
+  cluster_->hosts_ = {HostSharedPtr{
```

Comment:
I hate that we have to update so many callsites when a change like this happens. Not sure if there is any good way of making it better. I suppose at least within a test file you could add a function called something like makeTestHost() which would clean some of this up. Up to you.

Reply:
The other option is a default argument, but I'm not sure if I'd call that a "good way of making it better" :)

Reply:
Got rid of many of them by making a common helper method.